src/components/input/autocomplete-text/consult.js

```diff
@@ -0,0 +1,21 @@
+import {Component} from 'react';
+import ComponentBaseBehaviour from '../../../behaviours/component-base';
+
+@ComponentBaseBehaviour
+class AutocompleteTextConsult extends Component {
```

Comment:
It would be better to use a pure component rather than a class here, since it's stateless.